LICENSE

```diff
@@ -0,0 +1,23 @@
+MIT License
+
+Copyright 2017 Mystic Pants
```

Comment:
It should have Electric Imp copyright here.

Reply:
Ok, I made it Mystic Pants because that's what the submission guidelines say to do: https://electricimp.com/docs/libraries/submissions/